app/controllers/api/databases.php

```diff
@@ -2852,15 +3094,38 @@ function updateAttribute(
     ->label('abuse-key', 'ip:{ip},method:{method},url:{url},userId:{userId}')
     ->label('abuse-limit', APP_LIMIT_WRITE_RATE_DEFAULT * 2)
     ->label('abuse-time', APP_LIMIT_WRITE_RATE_PERIOD_DEFAULT)
-    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY, APP_AUTH_TYPE_JWT])
-    ->label('sdk.namespace', 'databases')
-    ->label('sdk.method', 'createDocument')
-    ->label('sdk.description', '/docs/references/databases/create-document.md')
-    ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
-    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
-    ->label('sdk.response.model', Response::MODEL_DOCUMENT)
-    ->label('sdk.offline.model', '/databases/{databaseId}/collections/{collectionId}/documents')
-    ->label('sdk.offline.key', '{documentId}')
+    ->label(
+        'sdk',
+        new Method(
+            namespace: 'databases',
+            name: 'createDocument',
+            description: '/docs/references/databases/create-document.md',
+            auth: [AuthType::SESSION, AuthType::KEY, AuthType::JWT],
+            responses: [
+                new SDKResponse(
+                    code: Response::STATUS_CODE_CREATED,
+                    model: Response::MODEL_DOCUMENT,
+                )
+            ],
+            responseType: ResponseType::JSON,
+            offlineKey: '{documentId}',
+            offlineModel: '/databases/{databaseId}/collections/{collectionId}/documents',
+            multiplex: [
+                new Multiplex(
+                    name: 'createDocument',
+                    parameters: ['documentId', 'data', 'permissions'],
+                    required: ['documentId', 'data'],
+                    responseModel: Response::MODEL_DOCUMENT,
+                ),
+                new Multiplex(
+                    name: 'createDocuments',
+                    parameters: ['documents'],
+                    required: ['documents'],
+                    responseModel: Response::MODEL_DOCUMENT_LIST,
+                ),
+            ]
+        )
```

Comment:
It would make more sense here if `sdk` has an array with two instances of `Method`